jsonschema-core.xml

```diff
@@ -2777,22 +2776,21 @@
                         </artwork>
                     </figure>
                     <t>
-                    Note that this pointer may not be resolvable by the normal JSON Pointer process
-                    due to the inclusion of these by-reference applicator keywords.
+                        Note that this pointer may not be resolvable by the normal JSON Pointer process
+                        due to the inclusion of these by-reference applicator keywords.
                     </t>
                     <t>
                         The JSON key for this information is "evaluationPath".
                     </t>
                 </section>
 
-                <section title="Keyword Absolute Location">
+                <section title="Subschema Location">
                     <t>
-                        The absolute, dereferenced location of the validating keyword.  The value MUST
-                        be expressed as a full IRI using the canonical IRI of the relevant schema resource
-                        with a JSON Pointer fragment, and it MUST NOT include by-reference applicators
-                        such as "$ref" or "$dynamicRef" as non-terminal path components.
-                        It MAY end in such keywords if the error or annotation is for that
-                        keyword, such as an unresolvable reference.
+                        The absolute, dereferenced location of the schema resource that produced
+                        the output unit.  The value MUST be expressed using the canonical IRI of the
+                        relevant schema resource plus a JSON Pointer fragment that indicates the contained
+                        subschema that produced the output.  It MUST NOT include by-reference applicators
```

Comment:
For consistency, can we replace "contained subschema" with "schema object" here as well?  In general I find myself using "schema object" instead of "subschema" as it works regardless of the location of the object.